exchange/exchange_test.go

```diff
@@ -1772,7 +1777,27 @@ func newExchangeForTests(t *testing.T, filename string, expectations map[string]
 		categoriesFetcher:   categoriesFetcher,
 		bidderInfo:          bidderInfos,
 		externalURL:         "http://localhost",
+		bidIDGenerator:      bidIDGenerator,
+	}
+}
+
+type mockBidIDGenerator struct {
+	GenerateBidID bool `json:"GenerateBidID"`
+	ReturnError   bool `json:"ReturnError"`
+}
```

Comment:
I had intended for the json names to use a lower case starting letter to match the rest of our existing json names:
```
type mockBidIDGenerator struct {
	GenerateBidID bool `json:"generateBidID"`
	ReturnError   bool `json:"returnError"`
}
```

Reply:
Oh I see, will do!

Reply:
Thank you.